src/main/kotlin/HelloWorld/FirstAPP.kt

```diff
@@ -0,0 +1,6 @@
+package HelloWorld
```

Comment:
The issue asks you to create a file `HelloWorld.kt` (not *FirstAPP.kt`). Please change this.

You can keep the package if you want but then please lowercase it, i.e. `helloworld`. That's not required but common practice.

:bulb: When renaming files, make sure to add both the addition of the new file _and_ the deletion of the old file to the index (with `git add`) before committing. Otherwise Git only learns about the new file and thinks there are two.